### DISCUSSION TITLE
Difference between negotiation `cancel` and `decline`

### DISCUSSION THREAD

**Discussion by ndr-brt** [eclipse-edc/Connector#2261] [Q&A]
The `ContractNegotiationService` offers two different methods with a similar scope:
- `decline`: puts the negotiation in `DECLINED` state
- `cancel`: puts the negotiation in `ERROR` state

The question is: are they overlapping? What's the point of having a method to put the negotiation in `ERROR`? 
The error state should be related to an exceptional error, not by an human will.

**Comment by jimmarino** [comment]
In the new IDS protocol specification, there will be only one state, TERMINATED. If we get rid of the error state, we should include a counter so that retry attempts are at some point abandoned and the negotiation terminated. Since an early draft of the spec should be available in December, maybe we can address all of this at the same time?

**Reply by ndr-brt** [reply]
I'm ok on discussing that later, my question was if that's correct to give to a user the possibility to move a negotiation to the `ERROR` state. I agree that it should exist, but it should be used only when the negotiation cannot happen for unexpected errors right?
My proposal would be to remove the `/cancel` endpoint/command
